### PR TITLE
template: several frontend fixes

### PR DESCRIPTION
**Pull request by crepererum**
This fixes several issues I've found while playing around with our default fronend.

Rebaised against: `master`

### CODE REVIEW

File: invenio/modules/search/templates/search/collection_base.html [hunk 1]
```diff
@@ -34,14 +34,18 @@
 <style>
 .websearch .nav > li > ul > li {
```
Comment (jirikuncar) [inline invenio/modules/search/templates/search/collection_base.html]: can you also move it to separate `search/collection.css`?

File: invenio/modules/communities/templates/communities/mycommunities_base.html [hunk 2]
```diff
@@ -19,9 +19,13 @@
 
 <div class="well">
 {% if current_user.is_authenticated() %}
-<a href="{{url_for('.new')}}" class="btn btn-primary pull-right"><i class="icon-plus-sign"></i> New</a>
 {% if mycommunities %}
-<h2>My communities</h2>
+<h2>
+  My communities
```
Comment (jirikuncar) [inline invenio/modules/communities/templates/communities/mycommunities_base.html]: if you are touching this file can you make all string translatable?

```jinja2
{{ _('My communities') }}
```

File: invenio/modules/communities/templates/communities/index_base.html [hunk 3]
```diff
@@ -30,6 +30,12 @@
 <div class="row">
 
   <div class="col-xs-12 col-sm-12 col-md-12">
+    <div class="page-header">
+      <h2>
+        <i class="fa fa-group"></i>
+        {{ _("Communities") }} <small>{{ _("created and curated by ") + config.CFG_SITE_NAME + _(" users") }}</small>
```
Comment (jirikuncar) [inline invenio/modules/communities/templates/communities/index_base.html]:
```
{{ _("created and curated by %(x_name)s users", x_name=config.CFG_SITE_NAME) }}
```

File: invenio/base/templates/header_base.html [hunk 4]
```diff
@@ -52,7 +52,7 @@
                   </ul>
                 </li>
             {%- else -%}
-            <li><a href="{{ item.url }}">{{ item.text|safe }}</a></li>
+            <li{% if (item.url|length > 1 and request.path.startswith(item.url)) or request.path == item.url %} class="active"{% endif %}><a href="{{ item.url }}">{{ item.text|safe }}</a></li>
```
Comment (jirikuncar) [inline invenio/base/templates/header_base.html]: we should better use `item.active` (from Flask-Menu).

File: invenio/modules/communities/templates/communities/index_base.html [hunk 5]
```diff
@@ -30,6 +30,12 @@
 <div class="row">
```
Comment (lnielsen) [inline invenio/modules/communities/templates/communities/index_base.html]: communities was kind of already transferred to inveniosoftware/invenio-communities? Perhaps we just remove communities from Invenio instead?

Comment (jirikuncar) [inline invenio/modules/communities/templates/communities/index_base.html]: I will move the commit there later. Shall we already remove communities from 2.1?

Comment (lnielsen) [inline invenio/modules/communities/templates/communities/index_base.html]: Yes, let's do that. It's only Zenodo really using them.

Comment (jirikuncar) [inline invenio/modules/communities/templates/communities/index_base.html]: Acked

**Comment by crepererum** [comment]
Ready to go

**Pull request merged**